packages/frontend-core/src/utils/utils.ts

```diff
@@ -45,9 +54,9 @@ export const sequential = fn => {
  * @param minDelay the minimum delay between invocations
  * @returns a debounced version of the callback
  */
-export const debounce = (callback, minDelay = 1000) => {
-  let timeout
-  return async (...params) => {
+export const debounce = (callback: Function, minDelay = 1000) => {
+  let timeout: NodeJS.Timeout
```

Comment:
NAB: Since this normally runs in the browser, this will actually just be a number. Sam and I ran into this same problem, where TS thinks this is running in a node env rather than a browser one.

The solution I've been using (until we find a proper fix to let TS know the correct environment) is just type this as `ReturnType<typeof setTimeout>`.

(I don't think this really affects anything here, but it's just more correct).